bundles/org.eclipse.ui.workbench/eclipseui/org/eclipse/ui/internal/Workbench.java

```diff
@@ -2699,7 +2704,8 @@ private static String buildCommandLine(String workspace) {
 	 * as the workspace location.
 	 *
 	 * @param workspacePath the new workspace location
-	 * @return {@link IApplication#EXIT_OK} or {@link IApplication#EXIT_RELAUNCH}
+	 * @return {@link IApplication#EXIT_OK} or {@link IApplication#EXIT_RELAUNCH} or
+	 *         <code>null</code>
```

Comment:
`null` breaks logic in `org.eclipse.ui.internal.ide.application.IDEApplication.checkInstanceLocation(Shell, Map)`, however I see the `null` was also added in be51d14c6f825ce5f072dcbf771449620e789b6a and javadoc was not updated too. See https://github.com/eclipse-platform/eclipse.platform.ui/pull/1622 + https://github.com/eclipse-platform/eclipse.platform.ui/pull/1622.